src/main/java/com/ldtteam/structures/client/BlueprintTessellator.java

```diff
@@ -86,12 +86,12 @@ private static void preBlueprintDraw()
         GlStateManager.texCoordPointer(LIGHT_TEX_COORD_COMPONENT_SIZE, GL_SHORT, VERTEX_SIZE, LIGHT_TEXT_COORD_COMPONENT_OFFSET);
         OpenGlHelper.setClientActiveTexture(OpenGlHelper.defaultTexUnit);
 
-        GlStateManager.disableCull();
+        // GlStateManager.disableCull();
```

Comment:
code commented out?

Reply:
No need to disable culling

Reply:
okay, just making sure you didn't accidentally leave it in from testing something